Extensions/quick_tags.js

```diff
@@ -67,7 +67,8 @@ XKit.extensions.quick_tags = new Object({
 
 	run: function() {
 		this.running = true;
-		if ($(".post.post_full").length <= 0) {return; }
+
+		if ($(".post.post_full").length <= 0 || $(".post.post_brick").length <= 0) { return; }
```

Comment:
Sorry, missed this the first time around. This should be ".post.post_full, .post.post_brick", right? as a selector, rather then OR-ing two selectors together